content/en/blog/2020/mosn-proxy/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Introducing MOSN: An Alternative Data Plane"
+subtitle: "A Cloud Native Proxy for Edge or Service Mesh"
+description: "An alternative sidecar proxy for Istio."
+publishdate: 2020-07-17
+attribution: "Wang Fakang(mosn.io)"
+keywords: [mosn,sidecar,proxy]
+---
+
+Thanks to the efforts of the MOSN community, make MOSN has completed the most of adaptation for Istio. MOSN [v0.14.0 release](https://github.com/mosn/mosn/releases/tag/v0.14.0) is now compatible with Istio 1.5.x, and it has gone through the Bookinfo sample as the data plane of Istio.
+
+## Background
+
+In the service mesh world, using Istio as the control plane has become the mainstream. Istio provides dynamic configuration of routes, service discovery, etc. to data plane proxies via the xDS protocol. The proxy can conveniently serve as Istio's data by simply interfacing with Envoy's xDS protocol. Istio's integration of third-party data planes can be implemented in three steps, as follows.
+
+- Implement xDS protocols to fulfill the capabilities for data plane related services.
+- Build `proxyv2` images using Istio's script and set the relevant `SIDCAR` and other parameters.
```

Comment:
SIDECAR ?

Reply:
`SIDECAR` is a environment variable, we can use it to set another proxy when build `proxyv2` image.
refer: https://github.com/istio/istio/pull/25146